app/Exceptions/InvalidSubscriptionTypeException.php

```diff
@@ -0,0 +1,34 @@
+<?php
+declare(strict_types=1);
+
+
+namespace App\Exceptions;
+
+use RuntimeException;
+use Throwable;
+
+class InvalidSubscriptionTypeException extends RuntimeException
+{
+    /**
+     * InvalidSubscriptionTypeException constructor.
+     *
+     * @param string $message
+     * @param int $code
+     * @param \Throwable|null $previous
+     */
+    public function __construct(String $message, Int $code, ?Throwable $previous = null)
```

Comment:
- you can initialize the value unto the constructor parameter as $code = 0.
- alphasort required variables